examples/platform/silabs/Rpc.cpp

```diff
@@ -104,27 +104,36 @@ class Efr32Device final : public Device
             ChipLogProgress(NotSpecified, "Did not receive a reboot delay. Defaulting to %d ms",
                             static_cast<int>(kRebootTimerPeriodMs));
         }
-        mRebootTimer = xTimerCreateStatic("Reboot", pdMS_TO_TICKS(delayMs), false, nullptr, RebootHandler, &mRebootTimerBuffer);
-        xTimerStart(mRebootTimer, 0);
+
+        mRebootTimer        = osTimerNew(RebootHandler, osTimerOnce, nullptr, &mRebootTimerAttr);
+        uint32_t delayTicks = ((uint64_t) osKernelGetTickFreq() * delayMs) / 1000;
```

Comment:
Should this use the pd_to_ms macro?

Reply:
Macro doesn't exist in cmsios. This way it abstract any underlying os cmsis uses

Reply:
hmm didn't know!